src/server_manager/web_app/ui_components/outline-server-view.js

```diff
@@ -686,6 +696,15 @@ export class ServerView extends DirMixin(PolymerElement) {
       this.accessKeySortDirection = 1;
       /** @type {(msgId: string, ...params: string[]) => string} */
       this.localize = null;
+      this.currentPage = '';
+      this.selectedTab = 'connections';
+    }
+
+    selectPage(page) {
```

Comment:
Instead of providing a method, let's keep using the `currentPage` assignment. (perhaps rename it to `selectedPage`). You are not preventing that direct assignment anyway.

A better approach is to listen on the `selected-changed` event and stop the animation there.
https://www.webcomponents.org/element/@polymer/iron-pages/elements/iron-pages#event-selected-changed